api/admin.py

```diff
@@ -1,11 +1,23 @@
 from django.contrib import admin
 
 from api.models import Pattern, SquadToken, Submission
-from api.tasks import submit_to_lava
+from api.tasks import submit_to_lava, check_job_status
+
+
+def check_status(modeladmin, request, queryset):
+    for pattern in queryset:
+        check_job_status.delay(pattern, {
+            'job': pattern.lava_job_id,
+            'description': 'something',
+            'status': "Submitted",
+            'pipeline': True,
```

Comment:
the issue is that check_job_status assumes it's receiving the data that we get from ZMQ ... I will have to think of something else.